scripts/build-docker.sh

```diff
@@ -1,5 +1,5 @@
 
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
should be on line 1. do we prefer /bin/bash or the /usr/bin/env bash?

Reply:
`#!/usr/bin/env bash` is the most portable `bash` shebang; and it should be preferred unless there is an explicit reason not to (I'm unaware of a general one).

Reply:
And yes, it should be on line-1.